service/worker/scheduler/workflow_test.go

```diff
@@ -1402,3 +1405,80 @@ func (s *workflowSuite) TestLimitedActions() {
 	s.True(s.env.IsWorkflowCompleted())
 	// doesn't end properly since it sleeps forever after pausing
 }
+
+// To generate workflow history for a specific server version, run the server with that version
+// uncomment the first line to run the workflow with the given schedule, and run the test. Make sure
+// to set the timeout flag accordingly. When test is finished, run temporal client with the command at the end
+// to capture the JSON history.
+func (s *workflowSuite) TestGenerateWorkflowHistoryForReplay() {
```

Comment:
this isn't really a test, and depends on a server running.. it could just be in a .go file and have a comment below saying to `go run` it